components/TodoList.js

```diff
@@ -163,36 +160,48 @@ function TodoList() {
                       <input
                         type="text"
                         value={editedTodoText}
-                        onChange={(e) => {
-                          if (e.target.value.length <= 12) {
-                            setEditedTodoText(e.target.value);
-                          }
-                        }}
+                        onChange={(e) => setEditedTodoText(event.target.value)}
```

Comment:
maybe you should not name your parameter of your arrow function like "e" in the future it could be helpful if you named it like "id", when your parameter is any id, or "slug" etc.